integration/spark/src/test/scala/org/apache/carbondata/spark/testsuite/iud/UpdateCarbonTableTestCase.scala

```diff
@@ -794,7 +795,31 @@ class UpdateCarbonTableTestCase extends QueryTest with BeforeAndAfterAll {
     sql("drop table if exists senten")
   }
 
-  test("block updating table which has index") {
+  // block update operation from table which has index. see PR2483
+  test("update should fail if the table has index") {
```

Comment:
Duplicate testcase.
Please check the testcase of 'update operation is not supported for index'

Reply:
ok